core/livepeernode.go

```diff
@@ -78,24 +88,28 @@ func NewLivepeerNode(e eth.LivepeerEthClient, vn net.VideoNetwork, nodeId NodeID
 		return nil, ErrLivepeerNode
 	}
 
-	return &LivepeerNode{VideoCache: NewBasicVideoCache(vn), VideoNetwork: vn, Identity: nodeId, Addrs: addrs, Eth: e, WorkDir: wd, PeerConns: make([]PeerConn, 0)}, nil
+	return &LivepeerNode{VideoCache: NewBasicVideoCache(vn), VideoNetwork: vn, Identity: nodeId, Addrs: addrs, Eth: e, WorkDir: wd}, nil
 }
 
 //Start sets up the Livepeer protocol and connects the node to the network
-func (n *LivepeerNode) Start(ctx context.Context, bootID, bootAddr string) error {
+func (n *LivepeerNode) Start(ctx context.Context, bootIDs, bootAddrs []string) error {
+	if len(bootIDs) != len(bootAddrs) {
+		return errors.New("BootIDs and BootAddrs do not match.")
+	}
 	//Set up protocol (to handle incoming streams)
 	if err := n.VideoNetwork.SetupProtocol(); err != nil {
 		glog.Errorf("Error setting up protocol: %v", err)
 		return err
 	}
 
 	//Connect to bootstrap node.  This currently also kicks off a bootstrap process, which periodically checks for new peers and connect to them.
```

Comment:
Should we update this comment since we're not periodically checking for new peers right now (or maybe mark it as a TODO)?

Reply:
Agreed, although IMO such a TODO for periodic checking probably better belongs within basicnet itself